script.js

```diff
@@ -0,0 +1,27 @@
+function check(number) {
+    if(number=== "" || isNaN(number)) {
+        alert("Prosze wpisać wartość liczbowa");
+        return false;
+    } else
+    return true;
+}
+function celcjusz()
+{
+	let number=document.getElementById('number').value;
+	let celcjusz=number*'1.8'+'32';
+	let result=document.getElementById('result');
+	if(check(number)){
+		result.innerHTML=number+" stopnie celcjusza to "+celcjusz+" fehrenheitów";
+	}
+
+}
+function fahrenheit()
+{
+	let number=document.getElementById('number').value;
```

Comment:
you use `document.getElementById('number').value` second time.

Try to create function that returns this and then use it like that

```suggestion
	let number= getInputValue()
```